packages/projection/src/operators/Mappers/withHandles.ts

```diff
@@ -95,6 +141,11 @@ const getBurnedHandles = (
       const handleData = tryCreateHandleOwnership(assetId, policyIds, cip67Assets, logger);
       if (handleData) {
         handles[handleData.handle] = handleData;
+        const parentHandle = getHandleParentAddress(handleData, logger);
+
+        if (parentHandle) {
+          handles[handleData.handle].parentHandle = parentHandle;
```

Comment:
This seems to be missing test coverage

Reply:
If this should work as it did before, I understand it's no longer needed so addressed here: 267c5f6a7d3fd5cdf5da332a666789470a7abf77

Reply:
should be addressed: b2fc531cece813e1ea89db59d3ab90cfa6204bc2